src/components/InputDropdown/styles.ts

```diff
@@ -0,0 +1,143 @@
+import styled, { keyframes } from 'styled-components';
+import { StylesConfig } from 'react-select';
+import COLORS from '@/styles/colors';
+import { DropdownOption } from '@/types/dropdown';
+
+export const DropdownWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+const fadeInKeyframes = keyframes`
+  from {
+    opacity: 0;
+    z-index: -1000;
+    transform: translateY(-0.5rem);
+  }
+  to {
+    opacity: 1;
+    z-index: 999;
+    transform: translateY(0);
+  }
+`;
+
+export const AnimatedWrapper = styled.div`
+  position: absolute;
+  top: 100%;
+  width: 100%;
+  background: white;
+  animation: 80ms ${fadeInKeyframes} cubic-bezier(0, 0, 0.35, 1);
+`;
+
+const getControlBorderColor = (
+  hasValue: boolean,
+  inFocus: boolean,
+  error?: boolean,
+) => {
+  if (error) return COLORS.redMid;
+  if (inFocus) return COLORS.blueMid;
+  return hasValue ? COLORS.greyDarker : COLORS.greyLight;
+};
+
+export const DropdownStyles = (
+  multi?: boolean,
+  error?: boolean,
+): StylesConfig<DropdownOption> => ({
+  noOptionsMessage: baseStyles => ({
+    ...baseStyles,
+    padding: '0.3125rem 0',
+  }),
+  dropdownIndicator: (baseStyles, state) => ({
+    ...baseStyles,
+    transform: state.selectProps.menuIsOpen ? 'rotate(180deg)' : '',
+    transition: '200ms ease-in-out',
+    color: COLORS.greyDarker,
+  }),
+  clearIndicator: (baseStyles, state) => ({
```

Comment:
can we make it so that when the button is clicked to clear the field, the dropdown menu does not automatically open?

Reply:
^^ and the same for the option tags!

Reply:
ye, it's doing that because i set `openMenuOnFocus` to be true... if it's false though, the menu won't show up when the user focuses on it with a keyboard. will look for a workaround, but if not ill set it to false (lawyers prolly wont use keyboard to navigate anyways)

Reply:
also a design question so cc @davidqing6432

Reply:
saw that you removed `openMenuOnFocus`, so will resolve this! but if you still want to run the keyboard stuff by david i'm making a [comprehensive list of design questions](https://www.notion.so/calblueprint/DESIGN-QUESTIONS-9bf3f8655ca4440ba5e2b62ff694fe4e), feel free to add

Reply:
thinking about it, i think it's ok to only show menu when the user types something. it might get annoying if the menu pops up every time if they want to navigate to another input.

Reply:
yeah i think its fine to just not open it up immediately